python/device/device.py

```diff
@@ -809,29 +817,31 @@ def handle_pingback_response_thread(self, worker_thread_info):
                 continue
 
             try:
-                msg = self.incoming_pingback_response_queue.get(timeout=1)
+                msg = self.incoming_service_ack_response_queue.get(timeout=1)
             except queue.Empty:
                 msg = None
 
             if msg:
                 arrivals = []
                 thief = json.loads(msg.data.decode())["thief"]
 
-                with self.pingback_list_lock:
-                    for pingback in thief["pingbacks"]:
-                        pingback_id = pingback["pingbackId"]
+                with self.service_ack_list_lock:
+                    for service_ack in thief["serviceAcks"]:
```

Comment:
consider making a constants file with things like "servcieAcks" to prevent bugs due to misspelled strings

Reply:
yes.  future PR.